tests/dialects/test_mysql.py

```diff
@@ -41,12 +41,6 @@ def test_binary_literal(self):
                 "mysql": "SELECT `0xz`",
             },
         )
-        self.validate_all(
-            "SELECT 0XCC",
-            write={
-                "mysql": "SELECT 0 AS XCC",
-            },
-        )
```

Comment:
In the previous code "SELECT 0xGM" would treat 0xGM as a literal and not like "SELECT 0 as xGM". Should scan_hex be smarter like scan_number to handle these cases?

Reply:
why did this test change?

Reply:
Discussed in slack. Now that 0XCG converts to a literal, the test is redundant now with the preceding test.